cg/nuget/cgmanifest.json

```diff
@@ -50,7 +50,7 @@
       "Component": {
         "Type": "NuGet",
         "NuGet": {
-          "Name": "Bond.Core.CSharp",
+          "Name": "Bond.CSharp",

```

Comment:
Presumably you used your dev build to generate this file, which is why it's different from what's checked in.

To avoid "Primary BuildXL PR Validation" failing, use the LKG version to build (which should generate the same content as the one currently checked in) or simply revert this file.